resources/views/index.blade.php

```diff
@@ -0,0 +1,45 @@
+@extends('layout')
+@section('content')
+    @if ($message = Session::get('success'))
+            <div class="alert alert-success">
+                <p>{{$message}}</p>
+            </div>
+    @endif
+    <div class="row">
+        <div class="col-md-6">
+            <h1>CRUD</h1>
+        </div>
+        <div class="col-md-6 text-right">
+            <a href="{{action('PostController@create')}}" class="btn btn-primary">Add Data</a>
+        </div>
+    </div>
+
+    <table class="table table-bordered">
+        <thead>
+            <tr>
+                <th>No</th>
+                <th>Name</th>
+                <th>Detail</th>
+                <th>Author</th>
+
+            </tr>
+        </thead>
+
+        <tbody>
+            @foreach ($posts as $post)
+                <th>{{$post->id}}</th>
+                <th>{{$post->name}}</th>
+                <th>{{$post->detail}}</th>
+                <th>{{$post->author}}</th>
```

Comment:
```suggestion
                <th>{{ $post->id }}</th>
                <th>{{ $post->name }}</th>
                <th>{{ $post->detail }}</th>
                <th>{{ $post->author }}</th>
```
khong dung id . dung index đi